crates/db_views_actor/src/community_view.rs

```diff
@@ -176,7 +176,7 @@ impl<'a> CommunityQuery<'a> {
         .filter(community::deleted.eq(false));
     }
 
-    match self.sort.unwrap_or(SortType::Hot) {
+    match self.sort {
```

Comment:
Makes more sense to me have these optional in the builder, and move the unwrap_or_default to here.